src/bundle/Command/CronRunCommand.php

```diff
@@ -46,6 +47,30 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $cron->setExecutor(new Executor());
         $cron->setResolver($resolver);
 
-        $cron->run();
+        $reports = $cron->run();
+
+        while ($cron->isRunning()) {}
```

Comment:
This may consume 100% of CPU for the duration of the command (depending on `isRunning()` implementation). Make it `usleep` for at least `10000` (10ms).

Reply:
@Nattfarinn I tested it without `while` statement. But then the reports output is not available. Don't know how this could be solved by using `usleep` as it is not known how long process is running?!

Reply:
@homarx `usleep` won't help you get the reports, but `while` body without it will definitely eat all the CPU cycles on doing absolutely nothing. `usleep` will free CPU cycles for other tasks and 10ms looped delay won't be noticeable at all.

Reply:
@Nattfarinn usleep is added to while statement